### ISSUE TITLE
zrepl v0.3.0 build fails on CentOS 7

### ISSUE THREAD

**Issue by poettler-ric** [zrepl/zrepl#367]
Hello, I tried to update the packages for Fedora/CentOS. Everything except EPEL 7 builds fine. EPEL 7 (go version: 1.13.14) fails with the following message:
```
make ZREPL_VERSION=0.3.0 zrepl-bin
GO111MODULE=on go build -mod=readonly  -ldflags "-X github.com/zrepl/zrepl/version.zreplVersion=0.3.0" -o "artifacts/zrepl-linux-amd64"
go: gonum.org/v1/gonum@v0.7.0 requires
        gonum.org/v1/plot@v0.0.0-20190515093506-e2840ee46a6b requires
        github.com/fogleman/gg@v1.2.1-0.20190220221249-0403632d5b90: invalid pseudo-version: git fetch --unshallow -f origin in /builddir/go/pkg/mod/cache/vcs/68f39ed450731f4018ee308789f8e36cf8876a81d65be6a72eaa43bfa29c1a63: exit status 128:
        fatal: git fetch-pack: expected shallow list
```

If you let me know how you fixed that the last time (I am not the best go developer) I could send you a pull request the next time.

**Comment by problame** [comment]
Did you have a look at #366 ? @awehrfritz put a new zrepl.spec together that seems to work. I wonder why though, since from what I can tell, their and you .spec look pretty much the same...

**Comment by awehrfritz** [comment]
It seems the issue on RHEL 7 is an issue with an too old git version and Go 1.13 as indicated in these bug report: https://github.com/golang/go/issues/38373 and https://github.com/golang/go/issues/34092

The spec file referenced in https://github.com/zrepl/zrepl/issues/366 does not even consider RHEL 7 (i.e. it fails on a much earlier stage when configuring the build). I think we should first figure out a solution to this issue here, and can then consider adding the necessary pluming to the spec file in https://github.com/zrepl/zrepl/issues/366.

**Comment by awehrfritz** [comment]
The issue here is indeed due to a too old git version in RHEL 7. Adding the [IUS community repositories](https://ius.io/) and requiring git >= 2 fixes the issue. I have implemented this in the spec file provided in PR #369.

**Comment by poettler-ric** [comment]
So you can also add this repository during the build process on copr? Didn't know that.

**Comment by problame** [comment]
RPM builds are addressed by #369 for zrepl 0.3.1 although.

@poettler-ric I won't issue 0.3.0 RPMs.
Are you going to fix the copr repos as suggested by @awehrfritz  in the meantime?
(Users of 0.3.1 and forward will use the offical zrepl-provided RPM repos, and existing users are going to be polling the current COPR repo for the 0.3.0 update)

**Comment by poettler-ric** [comment]
@problame I'd rather redirect people to @awehrfritz 's copr repositories, since I don't have anything to do with those sources.
